documentation/src/main/asciidoc/index.asciidoc

```diff
@@ -2,60 +2,58 @@
 Written and maintained by the {brandname} community
 :icons: font
 
-Get answers and find help in the {brandname} {infinispanversion} doc set.
+++++
+<script async src="https://cse.google.com/cse.js?cx=017043654585294774899:cdrjpvj5awo"></script>
+<div class="gcse-search"></div>
+++++
```

Comment:
@tristantarrant is this search option cool? here's the public url: https://cse.google.com/cse?cx=017043654585294774899:cdrjpvj5awo 

it includes https://infinispan.org/docs/stable/* and /docs/dev/* in the search result. it works fairly well but I don't know if this is the best solution. another thing is that I created this with my personal gmail account, which might not be ideal.

Reply:
I am not sure what is wrong with this particular search box, but it doesn't seem to find stuff :)
For example, I tried searching for "auto-config", "InfinispanIndexManager", "Query" and did not get any interesting result.
OTOH, the one from https://cse.google.com/cse?cx=013887093884325579276:mkumderkeqo do bring me all those results

Reply:
"local-heap" is another term it does not find meaningful results. Also, it appears the operator guide is not being indexed by the search

Reply:
we should probably use an infinispan.org account for this